src/i18n.tsx

```diff
@@ -74,6 +77,10 @@ i18n
                             namePlaceholder: 'Владислав Иванов',
                             registerButton: 'Регистрация',
                         },
+                        confirmation: {
+                            title: 'Подтверждение номера',
+                            subtitle: 'Ввведите код из смс',
+                        },
```

Comment:
названия ключей должны быть такие, чтобы видя только их, можно было понять о чем это. Когда будем переводить на казахский и забудем что-то перевести, то хотя бы будет возможность понять что там было в оригинале.
```suggestion
                            phoneConfirmation: 'Подтверждение номера',
                            enterSmsCode: 'Ввведите код из смс',
                        },
```
Да и в jsx-коде с такими ключами понятнее, что юзер видит на странице

Reply:
++